server/list.js

```diff
@@ -235,8 +235,14 @@ function buildTreeFromData(rootParent, previousData, breadcrumb) {
     const {slug} = docsInfo[id]
     const nextCrumb = breadcrumb ? breadcrumb.concat({ id: rootParent, slug: parentInfo.slug }) : []
 
-    // recurse building up breadcrumb
-    memo.children[slug] = buildTreeFromData(id, previousData, nextCrumb)
+    if (!memo.children[slug]) {
+      // recurse building up breadcrumb
+      memo.children[slug] = buildTreeFromData(id, previousData, nextCrumb)
+    } else {
+      log.debug(`Folder ${parentInfo.name} contains duplicate resources with slug ${slug}`)
```

Comment:
How would you feel about this as a `warn` level log, since we shouldn't really encourage this case to persist?

Reply:
👍